notesapi/v1/views.py

```diff
@@ -50,7 +52,10 @@ def get_from_db(self, *args, **kwargs):  # pylint: disable=unused-argument
         if 'text' in params:
             query = query.filter(Q(text__icontains=params['text']) | Q(tags__icontains=params['text']))
 
-        return [note.as_dict() for note in query]
```

Comment:
Can we remove `as_dict()` method, as we have `NoteSerializer` now, also, we can replace it with NoteSerializers in DetailView too.